HTAN.model.csv

```diff
@@ -155,7 +155,7 @@ Ligation Condition,Name of ligase and condition for proximity ligation,,,,TRUE,S
 Biotin Enrichment,Whether biotin is used for enriching ligation product,"Yes, No",,,TRUE,Sequencing,,,
 DNA Input Amount,"Amount of DNA for library construction, in nanograms.",,,,TRUE,Sequencing,,,int
 Resolution,"Binning size used for generating contact matrix, in basepair.",,,,TRUE,Sequencing,,,
-Stripe Calling,"Tool used for identifying architectural stripe-forming, interaction hotspots.","MACS2, Other",,,TRUE,Sequencing,,,
+Stripe Calling,"Tool used for identifying architectural stripe-forming, interaction hotspots.","MACS2, Other",,,TRUE,Sequencing,,,list::-?\d+

```

Comment:
I think we need to call this `list like` rather than just `list`. Unclear if the combination of `list like::regex` is in production. @aditigopalan can you check with Amy Heiser (by Slack)

Reply:
I think we also need to declare that what comes after is a regex

eg `list :: regex search [HTAN][0-9]{1}_[0-9]{4}_[0-9]*` from the s[chematic confluence docs](https://sagebionetworks.jira.com/wiki/spaces/SCHEM/pages/2645262364/Data+Model+Validation+Rules#list)

Reply:
> I think we need to call this `list like` rather than just `list`. Unclear if the combination of `list like::regex` is in production. @aditigopalan can you check with Amy Heiser (by Slack)

Just checked, Mialy and Amy [confirmed](https://sagebionetworks.slack.com/archives/C01ANC02U59/p1713553998029869) that this combination is in production as well. 

> I think we also need to declare that what comes after is a regex
> 
> eg `list :: regex search [HTAN][0-9]{1}_[0-9]{4}_[0-9]*` from the s[chematic confluence docs](https://sagebionetworks.jira.com/wiki/spaces/SCHEM/pages/2645262364/Data+Model+Validation+Rules#list)
does `list like :: regex search -?\d+` look right to you?

Reply:
The validation rule looks good but going back to the issue it should be applied to the `Loop window` attribute not the `Stripe Calling` attribute